pkg/api/converterfromapi.go

```diff
@@ -191,6 +208,12 @@ func convertAgentPoolProfileToVLabs(api *AgentPoolProfile, p *vlabs.AgentPoolPro
 	p.VnetSubnetID = api.VnetSubnetID
 	p.SetSubnet(api.Subnet)
 	p.FQDN = api.FQDN
+	p.Secrets = []vlabs.KeyVaultSecrets{}
```

Comment:
I'm confused, isn't this only in vlabs in my CR.  That is what I meant to do. I didn't update the 20160330 types.